android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -106,6 +107,13 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
           !it.value.isNullOrEmpty() &&
           !it.value.contains(STRING_INTERPOLATION_PREFIX)
       }
+    paramDataActionParams =
+      actionParams.filter {
+        it.paramType == ActionParameterType.PARAMDATA &&
+          !it.value.isNullOrEmpty() &&
+          !it.value.contains(STRING_INTERPOLATION_PREFIX)
```

Comment:
I do not think this check is necessary. Can we use a different `ActionParameterType` for this implementation? This will avoid ambiguity.

Reply:
You can for instance use `ActionParameterType.UPDATABLE_IDS` for lack of a better name